src/main/java/core/gui/comp/table/HOTableModel.java

```diff
@@ -354,14 +355,14 @@ public void storeUserSettings(JTable table) {
 		}
 	}
 
-	static public String formatTime( Timestamp ts){
-		if (ts != null ) return DateFormat.getDateTimeInstance().format(ts);
+	static public String formatTime( HODateTime ts){
+		if (ts != null ) return ts.toLocaleDateTime();
 		return "";
 	}
 
-	static public long time2Int(Timestamp ts){
-		if ( ts!= null) return ts.getTime();
-		return 0l;
+	static public long time2Int(HODateTime ts){
```

Comment:
This could be pushed into `HODateTime` itself.

Reply:
changed:     static public long toEpochSecond(HODateTime ts)